akka-actor-typed/src/main/mima-filters/2.6.5.backwards.excludes/jfr.excludes

```diff
@@ -0,0 +1,2 @@
+# JDK9+ classes for JFR
+ProblemFilters.exclude[Problem]("akka.actor.typed.internal.jfr.*")
```

Comment:
I think we should put this in `2.6.x.backwards.excludes`, otherwise we'll see it each release...

Reply:
moved